model/lollipop_keys.ts

```diff
@@ -50,36 +50,56 @@ export const PendingLolliPopPubKeys = t.interface({
 });
 export type PendingLolliPopPubKeys = t.TypeOf<typeof PendingLolliPopPubKeys>;
 
-export const NotPendingLolliPopPubKeys = t.interface({
+export const BaseNotPendingLollipopPubKeys = t.interface({
```

Comment:
We should keep coherence at least in the same file.
There's always `LolliPop` on ther the other types...

```suggestion
export const BaseNotPendingLolliPopPubKeys = t.interface({
```

Reply:
done in https://github.com/pagopa/io-functions-lollipop/pull/13/commits/b5d5d231fff8ca9e7201d2acf445b0ddaca4a880